apps/prairielearn/src/server.js

```diff
@@ -1842,6 +1862,23 @@ export async function initExpress() {
     },
     (await import('./pages/instructorQuestions/instructorQuestions.js')).default,
   ]);
+  app.use('/pl/course/:course_id(\\d+)/ai_generate_question_audit', [
+    function (req, res, next) {
+      res.locals.navSubPage = 'questions';
```

Comment:
You can inline this in the usage of the navbar partial on individual pages.

Reply:
done in 3c8bafd1470728a8c79b488213de1dfae0aed79e and 064e889cbc14078de7f4c52dac6d5817dcffab24